build.bat

```diff
@@ -0,0 +1,2 @@
+@echo Off

```

Comment:
nit: could we add this and the `build.sh` file to the solution under `files` - we have all other files there. 

_I'd add a suggestion, but that part of the `.sln` file is unchanged._

Reply:
Thanks, will add